src/figureprinter/FigurePainter.java

```diff
@@ -31,7 +37,18 @@ public static void drawLeftUpTriangle(int length){
      */
     public static void drawRightUpTriangle(int length){
         // TODO : complete the method
-
+        for (int i = 0; i < length; i++)
+        {
+            for (int j = 1; j <= length - i; j++)
```

Comment:
Ճիշտ է աշխատում նաև միայն փոքր պայմանի դեպքում (առանց =)

for (int j = 1; j < (length - i); j++) {